core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.html

```diff
@@ -26,33 +26,37 @@
 
             <div class="flex justify-content-center w-full container">
                 <div class="flex flex-column gap-2 content">
-                    <dot-experiments-reports-chart />
+                    <dot-experiments-reports-chart
+                        [config]="chartConfig"
+                        [data]="vm.chartData"
+                        [loading]="vm.isLoading"
+                    />
 
                     <dot-experiments-details-table
                         [data]="detailData"
-                        [title]="'dot.experimental.reports.resume' | dm"
+                        [title]="'experiments.reports.resume' | dm"
                     >
                         <ng-template #headers>
                             <div class="flex col flex-wrap align-items-center">
-                                {{ 'dot.experimental.reports.variants' | dm }}
+                                {{ 'experiments.reports.variants' | dm }}
                             </div>
                             <div class="flex col flex-wrap align-items-center justify-content-end">
                                 {{ 'experiments.configure.traffic.split.name' | dm }}
                             </div>
                             <div class="flex col flex-wrap align-items-center justify-content-end">
-                                {{ 'dot.experimental.reports.pageview' | dm }}
+                                {{ 'experiments.reports.pageview' | dm }}
                             </div>
                             <div class="flex col flex-wrap align-items-center justify-content-end">
-                                {{ 'dot.experimental.reports.sessions' | dm }}
+                                {{ 'experiments.reports.sessions' | dm }}
                             </div>
                             <div class="flex col flex-wrap align-items-center justify-content-end">
-                                {{ 'dot.experimental.reports.clicks' | dm }}
+                                {{ 'experiments.reports.clicks' | dm }}
                             </div>
                             <div class="flex col flex-wrap align-items-center justify-content-end">
-                                {{ 'dot.experimental.reports.best-variant' | dm }}
+                                {{ 'experiments.reports.best-variant' | dm }}
                             </div>
                             <div class="flex col flex-wrap align-items-center justify-content-end">
-                                {{ 'dot.experimental.reports.improvement' | dm }}
+                                {{ 'experiments.reports.improvement' | dm }}
                             </div>
```

Comment:
What about an ngFor here?

Reply:
Yes, could be a NgFor. I don't know if @hmoreras takes that into consideration in the issue about connect details with real data (https://github.com/dotCMS/core/issues/24626).